midi_app_controller/gui/midi_status.py

```diff
@@ -85,38 +91,69 @@ class MidiStatus(QWidget):
     def __init__(self):
         super().__init__()
 
-        # Binds selection.
-        selected_binds = state_manager.selected_binds
-        self.current_binds = DynamicQComboBox(
-            selected_binds.name if selected_binds is not None else None,
-            state_manager.get_available_binds,
-            state_manager.select_binds,
-        )
+        state.load_state()
```

Comment:
Wouldn't it fit better in `StateManager` constructor? I don't see any reason to call `load_state()` each time the widget is created - the `StateManager` may have already been initialized.